docs/guides/cloud/introduction.mdx

```diff
@@ -4,153 +4,131 @@ sidebar_position: 10
+lets you prioritize recently failed specs so they run faster on their next run.
+The [Auto Cancellation](/guides/cloud/smart-orchestration#Auto-Cancellation)
+feature will cancel a whole test run on failure, saving time and resource usage.
+You can also [cancel in-progress runs](/guides/cloud/runs#Run-cancellation)
+manually from Cypress Cloud if you need to.
+
+### Integrate with your favorite tools
+
+#### Source Control providers
 
-Ensure rock-solid reliability by keeping failing code out of your
-[GitHub](https://github.com), [GitLab](https://gitlab.com) and
-[Bitbucket](https://bitbucket.org) repositories, with
-[status checks](/guides/cloud/integrations/github#Status-checks) that block
-commits from being merged until your Cypress tests are green. Surface test
+Cypress Cloud integrates with GitHub, GitLab, and Bitbucket to provide
+rock-solid reliability by keeping failing code out of your main branch. Use
+[status checks](/guides/cloud/integrations/github#Status-checks) to block
+commits from being merged until your Cypress tests are green, and surface test
 results directly in your PRs with
 [pull request comments](/guides/cloud/integrations/github#Pull-request-comments)
 that include test run statistics, specific failure details, and deep links to
-results in Cypress Cloud for fast debugging. Users with
-[Business or Enterprise plans](https://www.cypress.io/pricing/) can integrate
-securely with
-[Self-managed GitLab](https://docs.gitlab.com/ee/subscriptions/self_managed/)
-instances too.
+results in Cypress Cloud for fast debugging.
 
 <DocsImage
   src="/img/dashboard/github-integration/status-checks-per-spec.jpg"
   alt="Status checks per spec"
 />
 
-### Collaborate and organize
+[Business and Enterprise plans](https://www.cypress.io/pricing/) include secure
+integration with self-managed
+[GitHub](https://docs.github.com/en/enterprise-server@3.5/admin/overview/about-github-enterprise-server)
+and [GitLab](https://docs.gitlab.com/ee/subscriptions/self_managed/) instances
+too.
 
-We also integrate with two of the world's most popular collaboration tools:
-[Slack](https://slack.com/) and [Jira](https://www.atlassian.com/software/jira).
-Deliver test results with valuable additional context, directly into a dedicated
-Slack channel. Or for those on the
-[Team, Business or Enterprise plans](https://www.cypress.io/pricing/),
-[create bidirectionally linked Jira tickets](/guides/cloud/integrations/jira#Creating-a-Jira-issue-for-a-test-case)
-directly from specific test failures.
+#### Team Communication
+
+Get notified on test progress directly in your team's
+[Slack](/guides/cloud/integrations/slack) or
+[Microsoft Teams](/guides/cloud/integrations/teams) channels.
 
 <DocsImage
   src="/img/dashboard/slack-integration/cypress-slack-integration-channel-feed.jpg"
   alt="Cypress notification feed in Slack channel"
   width={560}
 />
 
-Lastly you can use our flexible adminstrative functions to configure Cypress
```

Comment:
We might be able to reword this but calling out flexibly and SSO seems important.

Reply:
added it back in on its own section